src/vs/vscode.proposed.d.ts

```diff
@@ -1021,6 +1011,7 @@ declare module 'vscode' {
 		readonly document: TextDocument;
 		readonly metadata: NotebookCellMetadata
 		readonly outputs: ReadonlyArray<NotebookCellOutput>;
+		readonly previousResult: NotebookCellPreviousExecutionResult | undefined;
```

Comment:
Is `NotebookCellExecutionSummary` a better name?

Reply:
I'd say yes, I prefer that.

Reply:
Should `previousResult` be `previousSummary`?

Reply:
To my `previous` seems a little confusing.
The assumption is that there's a `current` or `another` execution, when it isn't the case.

I.e. `latest/last` seem more suitable, but thats me. I'd prefer to remove such adjectives from the name.

Reply:
I'll go with `latestExecutionSummary`